docs/chapter1/intermediate/component-archetype/component-archetype-structure.md

```diff
@@ -1,41 +1,41 @@
 # Component Archetype Structure
 
-In the previous component archetype, we used `demo-server` to demo and test the React component inside of the component itself, which adds a risk of confusion.  To clarify, we removed `demo-server` for the component archetype to target the build, eslint check, and unit tests.
+In the previous component archetype, we used `demo-server` to demo and test the React component inside of the component itself, which adds a risk of confusion. To clarify, we removed `demo-server` for the component archetype to target the build, eslint check, and unit tests.
 
-At the same time, in the [new electrode components](/chapter1/intermediate/create-a-electrode-component.md), we demonstrate components using `<repo>/demo-app` where we import components from `<repo>/packages/<componentName>`. The latest electrode components can be generated by the latest electrode-generator.
+At the same time, in the [new electrode components](../create-a-electrode-component.md), we demonstrate components using `<repo>/demo-app` where we import components from `<repo>/packages/<componentName>`. The latest electrode components can be generated by the latest electrode-generator.
 
 The new structure of the Electrode React Component Archetype is shown below.
 
 ```markdown
```

Comment:
why is this using `markdown` syntax highlight? is there a `markdown` syntax highlight.

anyways, this plus your auto formatting has messed up the directory tree.